src-electron/db/query-command.js

```diff
@@ -581,8 +581,9 @@ ON ENDPOINT_TYPE_CLUSTER.CLUSTER_REF = CLUSTER.CLUSTER_ID
 WHERE ENDPOINT_TYPE_COMMAND.ENDPOINT_TYPE_REF IN (${endpointTypeIds})
 AND ENDPOINT_TYPE_COMMAND.ENDPOINT_TYPE_REF = ENDPOINT_TYPE_CLUSTER.ENDPOINT_TYPE_REF
 AND ENDPOINT_TYPE_CLUSTER.SIDE IN ("client", "server") AND ENDPOINT_TYPE_CLUSTER.ENABLED=1 AND ${isIncomingOrOutgoingSql}
-AND CLUSTER.NAME = "${clName}" AND ENDPOINT_TYPE_CLUSTER.SIDE = "${clSide}" 
-${mfgSpecificString} GROUP BY COMMAND.NAME`
+AND CLUSTER.NAME = "${clName}" AND ENDPOINT_TYPE_CLUSTER.SIDE = "${clSide}"
+${mfgSpecificString} GROUP BY COMMAND.NAME
+ORDER BY CLUSTER.CODE, COMMAND.CODE`
```

Comment:
I know zigbee pro and Matter differ on this a bit so it will be better to support both and go with
CLUSTER.MANUFACTURER_CODE, CLUSTER.CODE, COMMAND.MANUFACTURER_CODE, COMMAND.CODE

Reply:
Sure, feel free to make a suggested edit and commit it, that works for me, I just want some order.